Sources/WKSnapshotConfigurationShim/include/WKSnapshotConfigurationShim.h

```diff
@@ -0,0 +1,2 @@
+@import CoreGraphics;
+#import <WebKit/WKSnapshotConfiguration.h>
```

Comment:
This shim should ideally be scoped for iOS 11 and macOS 10.3. Not quite sure how to organize everything yet.